src/contact-details/age-helpers.tsx

```diff
@@ -0,0 +1,71 @@
+// Based on http://www.gregoryschmidt.ca/writing/patient-age-display-ehr-conventions,
+// which is different from npm packages such as https://www.npmjs.com/package/timeago
+export function age(dateString: string): string {
+  // First calculate the age in years
+  const today = new Date();
+  const birthDate = new Date(dateString);
+  const monthDifference = today.getUTCMonth() - birthDate.getUTCMonth();
+  const dateDifference = today.getUTCDate() - birthDate.getUTCDate();
+  let age = today.getUTCFullYear() - birthDate.getUTCFullYear();
+  if (monthDifference < 0 || (monthDifference === 0 && dateDifference < 0)) {
+    age--;
+  }
+
+  // Now calculate the number of months in addition to the year's age
+  let monthsAgo = monthDifference >= 0 ? monthDifference : monthDifference + 12;
+  if (dateDifference < 0) {
+    monthsAgo--;
+  }
+  const monthsAgoStr = monthsAgo > 0 ? `${monthsAgo} mo` : "";
```

Comment:
`monthsAgo > 0 ? '${monthsAgo} mo' : ""` we can use this like `monthsAgo > 0 && '${monthsAgo} mo'` right?

Reply:
No. It's not the same.